frame/nomination-pools/src/lib.rs

```diff
@@ -2479,8 +2479,8 @@ impl<T: Config> Pallet<T> {
 impl<T: Config> OnStakerSlash<T::AccountId, BalanceOf<T>> for Pallet<T> {
 	fn on_slash(
 		pool_account: &T::AccountId,
-		// Bonded balance is always read directly from staking, therefore we need not update
-		// anything here.
+		// Bonded balance is always read directly from staking, therefore we
```

Comment:
please make sure lines are wrapped at 100, not less.

Reply:
I use rewrap for wrapping all of my comments as you recommended, not sure why it sometimes fails to wrap at 100.